packages/app/src/utils/explorers.ts

```diff
@@ -158,12 +158,21 @@ export const EXPLORERS_CONFIG: Record<NETWORK, ExplorerData> = {
   [NETWORK.PLASMA_TESTNET]: {
     networkExplorerName: "Plasmascan",
     networkExplorerUrl: "https://testnet.plasmascan.to/",
-    networkExplorerApiUrl: "https://testnet.plasmascan.to/api",
-    safeTransactionApi: "https://transaction-plasma-testnet.stage.safe.protofire.io/api",
+    networkExplorerApiUrl: "https://api.routescan.io/v2/network/testnet/evm/9746_5/etherscan/api",
+    safeTransactionApi: "https://transaction-plasma-testnet.safe.protofire.io/api",
     safeUrl: "https://app.safe.protofire.io/plasma-testnet",
     verifyContractUrl: "https://testnet.plasmascan.to/verifycontract",
     explorerApiKey: process.env.REACT_APP_PLASMASCAN_KEY ?? '',
   },
+  [NETWORK.PLASMA]: {
+    networkExplorerName: "Plasmascan",
+    networkExplorerUrl: "https://plasmascan.to/",
+    networkExplorerApiUrl: "https://api.routescan.io/v2/network/mainnet/evm/9745/etherscan/api",
+    safeTransactionApi: "https://transaction-plasma.safe.protofire.io/api",
+    safeUrl: "https://app.safe.protofire.io/plasma",
```

Comment:
```suggestion
    safeUrl: "https://app.safe.protofire.io/plasma:",
```

This safeUrl will be concatenated with safe address and require colon